convert2rhel/utils.py

```diff
@@ -246,13 +248,13 @@ def backup_and_track_removed_pkg(self, pkg):
     def _remove_installed_pkgs(self):
         """For each package installed during conversion remove it."""
         loggerinst = logging.getLogger(__name__)
-        loggerinst.task("Rollback: Removing installed packages")
+        loggerinst.task("Rollback: Removing installed packages") # pylint: disable=W1201
```

Comment:
I wonder why is this needed here. There are no arguments passed. http://pylint-messages.wikidot.com/messages:w1201.
Actually, what pylint reported on the latest master on this line is E1103, not W1201:
https://github.com/oamg/convert2rhel/runs/709656220#step:5:44

The same applies to the change few lines below.

Reply:
I'll rebase, several files are changed since I send these PR.